src/main/java/lotto/domain/WinnigTicket.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+public class WinnigTicket {
```

Comment:
Ticket과 LottoNumber를 활용해 당첨 번호를 구현해 주셨네요 👍 
클래스명에 오타가 있는 것 같아요
```suggestion
public class WinningTicket {
```

Reply:
오타 수정하엿습니다.